examples/integration/test/fixtures/bwb.xcodeproj/rules_xcodeproj/external.xcfilelist

```diff
@@ -18,4 +18,7 @@ $(BAZEL_EXTERNAL)/com_google_google_maps/GoogleMapsCore.xcframework/ios-arm64_x8
 $(BAZEL_EXTERNAL)/examples_command_line_external/ExternalFramework.framework
 $(BAZEL_EXTERNAL)/examples_command_line_external/ImportableLibrary/Library.h
 $(BAZEL_EXTERNAL)/examples_command_line_external/ImportableLibrary/libImportableLibrary.a
+$(BAZEL_EXTERNAL)/examples_ios_app_external/Assets.xcassets
```

Comment:
Interesting that this appears now.

Reply:
Indeed, not idea if this is surfacing something we should investigate?

Reply:
Now that I think about it, it's probably just you adding `"//iOSApp/Test/UITests:iOSAppUITests"` to the targets. Can you do that as a stand alone PR that merges before this, to keep the noise down?

Reply:
Yep, definitely, will put up that PR in a few. Btw noticed that `//iOSApp/Test/UITests:iOSAppUITests` was also not set as top level target in the `rules_ios` example, should be put both in a separate PR then? Or one for each?

Reply:
I'll put both in separate commits, PR coming soon

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/2202